src/components/elements/Container.ts

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled'
+import theme from '../../assets/theme'
+import constants from '../../assets/theme/constants'
+
+const Container = styled('div')`
+  width: 94vw;
+  max-width: 1660px;
```

Comment:
The container width is now scalable but it should be static. If you intended to increase the size of the container above the `xl` breakpoint then you should use this value from constants object: 
```suggestion
  width: ${constants.containerSizes.xl};
```

---

Our container should always have fixed width values because we want to limit our content to the controlled areas where we always know the width. It helps to predict how our content would adapt to different resolutions.